### PR TITLE
Replaced rpy2 with Python STL port rstl

### PR DESCRIPTION
**Pull request by ericist**
Hey,
I replaced rpy2 with my Python STL port [rstl](https://github.com/ericist/rstl) so pyculiarity doesn't require an R installation anymore.
The time the tests take increased from 6 to 29 seconds on my machine, but I guess that is to be expected.

This also requires pull request https://github.com/nicolasmiller/pyculiarity/pull/21 by @michael-erasmus to work.

### CODE REVIEW

File: pyculiarity/detect_anoms.py [hunk 1]
```diff
@@ -67,18 +67,18 @@ def detect_anoms(data, k=0.49, alpha=0.05, num_obs_per_period=None,
         data = data.resample(resample_period)
 
 
-    decomp = stl(data.value, "periodic", np=num_obs_per_period)
+    decomp = STL(data.value, num_obs_per_period, "periodic", robust=True)
```
Comment (nicolasmiller) [inline pyculiarity/detect_anoms.py]: Is the robust argument in the original R / Fortran etc? Taking a look at the rstl code now to see what this does.

**Comment by nicolasmiller** [comment]
This is awesome, Eric. Thank you for doing this. I see that rstl is intended for Python3. Do you have a sense of if it's possible to run this entire thing under Python 2? I'd like to support both if we can.

File: pyculiarity/detect_anoms.py (hunk 1 above)
Comment (ericist) [reply pyculiarity/detect_anoms.py]: Yes, Twitter also passes robust=TRUE here:
https://github.com/twitter/AnomalyDetection/blob/1f5deaa1609f8f1964c1e905c7a8ad2d1d0dc718/R/detect_anoms.R#L42

**Comment by ericist** [comment]
Great, I'll look into Python 2 compatibility and let you know.

**Comment by ericist** [comment]
As it turns out the rstl code itself is already compatible with Python 2.
I just uploaded the Python 2 version to PyPi.

**Comment by ericist** [comment]
Okay, rstl should now be compatible with Python 2, turns out Python 2's math.ceil returns a float and Python 3's an int.

**Comment by nicolasmiller** [comment]
Fantastic. Thanks again, Eric. I should have some time to update the PyPI release in the next day or so.

**Pull request merged**